docs/release-notes/relnotes-2012.md

```diff
@@ -41,3 +41,25 @@
 - (GDO-544) Resolved issue where PingDataGovernance PAP images' MAX_HEAP_SIZE variable had no effect.
 - (GDO-618) Resolved issue where base layer was missing JMX agent.
 - (GDO-640) Resolved issue where wait-for command didn't honor timeout when waiting for host:port.
+
+### Product Build Matrix
+
+The following table includes product versions and their accompanying Image build status for this release.
+
+| Product | Active Build | Build EOL |
+|------|------|------|
+| PingAccess | <b>6.2.0</b><br/>6.1.3 | 6.0.4 |
+| PingCentral |<b>1.6.0</b><br/>1.5.0 |  |
+| PingDataConsole | <b>8.2.0</b><br/>8.1.0.0 | 8.0.0 |
```

Comment:
Why we use both 3 and 4 digit version numbers for the PingData products?

Reply:
it should all be 4

Reply:
Should be fixed in the latest push. I originally had the 4-digit versions in bullet forms and must have over deleted the bullet list.

Reply:
I like it.